scalding-serialization/src/main/scala/com/twitter/scalding/serialization/OrderedSerialization.scala

```diff
@@ -101,6 +101,30 @@ object OrderedSerialization {
     case NonFatal(e) => CompareFailure(e)
   }
 
+  def viaTransform[T, U](
+    packFn: T => U,
+    unpackFn: U => T)(implicit otherOrdSer: OrderedSerialization[U]): OrderedSerialization[T] =
+    new OrderedSerialization[T] {
+
+      override def hash(t: T) = otherOrdSer.hash(packFn(t))
```

Comment:
Thats not thread safe right? And we can't use the equiv from the other serialization since we'll need to compare the pre-transformed version ?